tests_integration/__tests__/__snapshots__/early-exit.js.snap

```diff
@@ -3,53 +3,121 @@
   --ignore-path <path>     Path to a file containing patterns that describe files to ignore.
                            Defaults to ./.prettierignore.
-  --stdin                  Read input from stdin.
-  --stdin-filepath         Path to the file used to read from stdin.
-  --print-width <int>      Specify the length of line that the printer will wrap on. Defaults to 80.
-  --tab-width <int>        Specify the number of spaces per indentation-level. Defaults to 2.
-  --use-tabs               Indent lines with tabs instead of spaces.
-  --no-semi                Do not print semicolons, except at the beginning of lines which may need them.
-  --single-quote           Use single quotes instead of double quotes.
-  --no-bracket-spacing     Do not print spaces between brackets.
+  --no-config              Do not look for a configuration file.
+  --with-node-modules      Process files inside 'node_modules' directory.
+
+Format options:
+
   --jsx-bracket-same-line  Put > on the last line instead of at a new line.
-  --trailing-comma <none|es5|all>
-                           Print trailing commas wherever possible when multi-line. Defaults to none.
+  --no-bracket-spacing     Do not print spaces between brackets.
+  --no-semi                Do not print semicolons, except at the beginning of lines which may need them.
   --parser <flow|babylon|typescript|postcss|json|graphql>
                            Specify which parse to use. Defaults to babylon.
-  --cursor-offset <int>    Print (to stderr) where a cursor at the given position would move to after formatting.
-                           This option cannot be used with --range-start and --range-end
+  --print-width <int>      Specify the length of line that the printer will wrap on. Defaults to 80.
+  --range-end <int>        Format code ending at a given character offset (exclusive).
+                           The range will extend forwards to the end of the selected statement.
+                           This option cannot be used with --cursor-offset.
+                           Defaults to Infinity.
   --range-start <int>      Format code starting at a given character offset.
                            The range will extend backwards to the start of the first line containing the selected statement.
                            This option cannot be used with --cursor-offset.
                            Defaults to 0.
+  --single-quote           Use single quotes instead of double quotes.
+  --tab-width <int>        Specify the number of spaces per indentation-level. Defaults to 2.
+  --trailing-comma <none|es5|all>
+                           Print trailing commas wherever possible when multi-line. Defaults to none.
+  --use-tabs               Indent lines with tabs instead of spaces.
+
+Uncategorized options:
+
+  --cursor-offset <int>    Print (to stderr) where a cursor at the given position would move to after formatting.
```

Comment:
I'd say the following are "directives"/"commands", as they completely change the behaviour of the program:

* `--list-different`
* `--find-config-path`
* `--help`
* `--version`